app/frontend/src/components/AvatarInput.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, Story } from "@storybook/react";
+import { useForm } from "react-hook-form";
+
+import AvatarInput from "./AvatarInput";
+import Button from "./Button";
+
+export default {
+  component: AvatarInput,
+  title: "Components/Compound/AvatarInput",
+} as Meta;
+
```

Comment:
This is broken when pressing confirm. I think `uploadFile` from service in this will need mocking.

Reply:
I used `ChangeEmail.stories.tsx` as a mock example but it isn't mocking... any idea why?

Reply:
Maybe it's because `AvatarInput` is importing `service` instead of `service/index`?